src/components/settings/GcodePreviewSettings.vue

```diff
@@ -270,6 +282,22 @@ export default class GcodePreviewSettings extends Vue {
       value,
       server: true
     })
+
+    if (!value) {
+      this.autoLoadMobileOnPrintStart = false
+    }
```

Comment:
It makes sense to me that `autoLoadOnPrintStart` should override `autoLoadMobileOnPrintStart`, so I've added this to ensure that mobile loading is disabled when `autoLoadOnPrintStart` is disabled